ex3.html

```diff
@@ -0,0 +1,558 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<link rel="stylesheet" type="text/css" href="ex1_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="ex2_header_css.css">
+	<link rel="stylesheet" type="text/css" href="ex3_bod_css.css">
+	<link rel="stylesheet" type="text/css" href="buttons.css">
+</head>
+<body>
+	<section>
+		<header class="header"> 
+			<img src="logo.jpg" style="float:left; width:200px;height:40px; margin-left: 3%">
+			<h2><a href="ex2.html" target="_self" class="button2" style="margin-left: 7%; margin-right: 0.1%;">Evaluations</a></h2>
+			<h2><a href="ex3.html" target="_self" class="button1"> New Evaluation</a></h2>
+			<h4><a href="ex1.html" target="_top" style="float: right; margin-right: 2%; font-size: 150%; text-decoration: none;">Logout</a></h4>
+		</header>
+	</section>
+
+	<section>
+		<div id="dm">
+			<form action="/action_page.php">
+			<input class="textbox" type="text" placeholder="Candidate" name="CandName">
+				<input class="textbox" type="text" placeholder="Interviewer" name="IntervName">
+				<input class="textbox" type="date" name="ddate">
+
+
+				<style type="text/css">
+					ul {
+						list-style-type: none;
+
+					}
+					li {
+						float: left;
+						margin-right: 20px;
+					}
+
+					fieldset {
+						margin-bottom: 50px;
+
+					}
+					textarea {
+						height: 100px;
+						resize: none;
+						width: 100%;
+						
+					}
+
+					#div1 {
+						height: 100px;
+
+						border:1px solid black;
+					}
+
+					#div2 {
+						height: 50px;
+
+						background-color: lightblue;
+					}
+
+					span {
+						margin-right: 250px;
+					}
+					#dm {
+						width: 1200px;
+						margin: 0 auto;
+					}
+
+					.resize {
+						height: 40px;
+						width: 32.8%;
+						border: 0;
+					}
+				</style>
+
+				<h1 class="boxTitle">Technical Level</h1>
+				<div id="div1">
+					<div id="div2">
+						<span> Trainee </span>
+						<span> Junior </span>
+						<span> Middle </span>
+						<span> Senior </span>
+					</div>
+
+
+					<input type="radio" name="b" value="Trainee" style="margin-left: 20px; margin-right: 250px">
+					<input type="radio" name="b" value="Junior1">
+					<input type="radio" name="b" value="Junior2">
+					<input type="radio" name="b" value="Junior3" style="margin-right: 225px">
```

Comment:
I, for one, would have wrapped each of these levels in a separate container, so that I can easily space them out via CSS, instead of inline.